Cargo.toml

```diff
@@ -7,7 +7,7 @@ members = [
 
 [package]
 name = "async-stripe"
-version = "0.37.2"
+version = "0.37.3"
```

Comment:
I don't think you need to manually bump this. This will probably be taken care when a new version is created from master.

Reply:
Correct (and thanks for reviewing @augustoccesar!) please revert this line. You should instead prefix your commit message with `fix:` and semantic versioning will handle the rest.